lib/mllparty/slack_notification.ex

```diff
@@ -0,0 +1,28 @@
+defmodule SlackNotification do
+  require HTTPoison
```

Comment:
@mfos239 require is only necessary if you're using macros from the require'd module